tests/ert/unit_tests/config/parsing/test_observations_parser.py

```diff
@@ -536,3 +536,22 @@ def test_that_multiple_segments_are_collected():
     )
 
     assert len(observations[0][1].segment) == 2
+
+
+def test_that_duplicate_keys_results_in_error_message_with_location():
+    with pytest.raises(
+        ObservationConfigError,
+        match=r"Line 7 \(Column 16-25\): Observation contains duplicate key ERROR_MIN",
+    ):
+        parse_content(
+            """
+            HISTORY_OBSERVATION GWIR:FIELD
```

Comment:
Suggest testing for `SUMMARY_OBSERVATION` and `GEN_OBS` as well, those are move widely used then `HISTORY_OBSERVATION`

Reply:
Added.